Assets/Plugins/RuntimeTransformHandle/Scripts/RuntimeTransformHandle.cs

```diff
@@ -37,6 +38,10 @@ public class RuntimeTransformHandle : MonoBehaviour
 
         public Transform target;
 
+        public UnityEvent startedDraggingHandle = new UnityEvent();
+        public UnityEvent draggingHandle = new UnityEvent();
```

Comment:
Het kan verwarrend zijn dat deze variabele dezelfde naam heeft als `_draggingHandle` en iets anders is; maar ik zou de PR er niet op afwijzen :)

Reply:
ik kan er isDraggingHandle van maken? _draggingHandle wil ik niet aanpassen omdat het de naam is van de variabele in de originele repo

Reply:
Klinkt goed!